src/solvers/refinement/string_functions.h

```diff
@@ -0,0 +1,79 @@
+const irep_idt cprover_string_insert_long_func("__CPROVER_uninterpreted_string_insert_long_func");
+const irep_idt cprover_string_insert_bool_func("__CPROVER_uninterpreted_string_insert_bool_func");
+const irep_idt cprover_string_insert_char_func("__CPROVER_uninterpreted_string_insert_char_func");
+const irep_idt cprover_string_insert_float_func("__CPROVER_uninterpreted_string_insert_float_func");
+const irep_idt cprover_string_insert_double_func("__CPROVER_uninterpreted_string_insert_double_func");
+const irep_idt cprover_string_insert_char_array_func("__CPROVER_uninterpreted_string_insert_char_array_func");
+const irep_idt cprover_string_is_prefix_func("__CPROVER_uninterpreted_string_is_prefix_func");
+const irep_idt cprover_string_is_suffix_func("__CPROVER_uninterpreted_string_is_suffix_func");
+const irep_idt cprover_string_is_empty_func("__CPROVER_uninterpreted_string_is_empty_func");
+const irep_idt cprover_string_last_index_of_func("__CPROVER_uninterpreted_string_last_index_of_func");
+const irep_idt cprover_string_length_func("__CPROVER_uninterpreted_string_length_func");
+const irep_idt cprover_string_data_func("__CPROVER_uninterpreted_string_data_func");
+const irep_idt cprover_string_of_int_func("__CPROVER_uninterpreted_string_of_int_func");
+const irep_idt cprover_string_of_int_hex_func("__CPROVER_uninterpreted_string_of_int_hex_func");
+const irep_idt cprover_string_of_long_func("__CPROVER_uninterpreted_string_of_long_func");
+const irep_idt cprover_string_of_bool_func("__CPROVER_uninterpreted_string_of_bool_func");
+const irep_idt cprover_string_of_float_func("__CPROVER_uninterpreted_string_of_float_func");
+const irep_idt cprover_string_of_double_func("__CPROVER_uninterpreted_string_of_double_func");
+const irep_idt cprover_string_of_char_func("__CPROVER_uninterpreted_string_of_char_func");
+const irep_idt cprover_string_of_char_array_func("__CPROVER_uninterpreted_string_of_char_array_func");
+const irep_idt cprover_string_parse_int_func("__CPROVER_uninterpreted_string_parse_int_func");
+const irep_idt cprover_string_replace_func("__CPROVER_uninterpreted_string_replace_func");
+const irep_idt cprover_string_set_length_func("__CPROVER_uninterpreted_string_set_length_func");
+const irep_idt cprover_string_startswith_func("__CPROVER_uninterpreted_string_startswith_func");
+const irep_idt cprover_string_substring_func("__CPROVER_uninterpreted_string_substring_func");
+const irep_idt cprover_string_to_char_array_func("__CPROVER_uninterpreted_string_to_char_array_func");
+const irep_idt cprover_string_to_lower_case_func("__CPROVER_uninterpreted_string_to_lower_case_func");
+const irep_idt cprover_string_to_upper_case_func("__CPROVER_uninterpreted_string_to_upper_case_func");
+const irep_idt cprover_string_trim_func("__CPROVER_uninterpreted_string_trim_func");
+const irep_idt cprover_string_value_of_func("__CPROVER_uninterpreted_string_value_of_func");
```

Comment:
This has been done in https://github.com/diffblue/cbmc/pull/341/commits/c81641afc803acc0011bc32ef84a969c282f7173
The string_functions.h file is now useless and has been removed.
This change also made necessary to deal with C string functions in a different way. They are now processed by pass_preprocess in a similar way to java functions